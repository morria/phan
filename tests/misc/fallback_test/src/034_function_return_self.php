<?php

/** @return self */
function h34() {
    return new stdClass();
}
