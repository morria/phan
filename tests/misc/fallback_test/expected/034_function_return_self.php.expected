src/034_function_return_self.php:3 PhanContextNotObjectUsingSelf Cannot use self as type when not in object context in \h34
